using-mobile-extensions/adobe-edge-identity/README.md

```diff
@@ -0,0 +1,258 @@
+
+1. In Experience Platform Launch, in your mobile property, click the **Extensions** tab.
+2. On the **Catalog** tab, locate or search for the **Edge Identity** extension, and click **Install**.
+3. There are no configuration settings for **Edge Identity**.
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration.
+
+## Add the AEP Edge Identity extension to your app
+
+### Download and import the Edge Identity extension
+{% hint style="info" %}
+The following instructions are for configuring an application with Edge Identity. If an application will include both Identity and Edge Identity extensions, follow the instructions [below](#download-and-import-the-identity-and-edge-identity-extensions).
+{% tabs %}
+
+{% tab title="Android" %}
+### Java
+
+1. Add the Mobile Core and Edge extensions to your project using the app's Gradle file.
+
+   ```java
+   implementation 'com.adobe.marketing.mobile:core:1.+'
+   implementation 'com.adobe.marketing.mobile:edge:1.+'
+   implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
+   ```
+   
+2. Import the Mobile Core and Edge extensions in your application class.
+
+   ```java
+    import com.adobe.marketing.mobile.*;
+   ```
+{% endtab %}
+
+{% tab title="iOS" %}
+1. Add the Mobile Core and Edge extensions to your project using CocoaPods. Add following pods in your `Podfile`:
+
+   ```swift
+   use_frameworks!
+   target 'YourTargetApp' do
+       pod 'AEPCore'
+       pod 'AEPEdge'
+       pod 'AEPEdgeIdentity'
+   end
+   ```
+   
+2. Import the Mobile Core and Edge libraries:
+
+### Swift
+
+```swift
+// AppDelegate.swift
+import AEPCore
+import AEPEdge
+import AEPEdgeIdentity
+```
+
+### Objective-C
+
+```text
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+```
+{% endtab %}
+{% endtabs %}
+
+### Register the Edge Identity extension with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+### Java
+
+```java
+public class MobileApp extends Application {
+
+    @Override
+    public void onCreate() {
+      super.onCreate();
+      MobileCore.setApplication(this);
+      MobileCore.configureWithAppID("yourLaunchEnvironmentID");
+
+      try {
+        Edge.registerExtension();
+        Identity.registerExtension();
+        MobileCore.start(new AdobeCallback() {
+          @Override
+          public void call(final Object o) {
+            // processing after start
+          }});
+      } catch (Exception e) {
+        //Log the exception
+      }
+    }
+}
+```
+{% endtab %}
+
+{% tab title="iOS" %}
+### Swift
+
+```swift
+// AppDelegate.swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    MobileCore.registerExtensions([Identity.self, Edge.self], {
+    MobileCore.configureWith(appId: "yourLaunchEnvironmentID")
+  })
+  ...
+}
+```
+
+### Objective-C
+
+```text
```

Comment:
can we try the `objectivec` tag not sure if it makes difference. But I presume its better than text

Reply:
Yeah, the objectivec tag will provide syntax highlighting for Obj-C.

Reply:
Updated to use the `objectivec` tag.